README.md

```diff
@@ -102,6 +100,54 @@ They can then be loaded using the `_filename` attribute:
 
 This is useful for long examples.
 
+The `param` elements contain a `pos` attribute, which indicates what capture field
+from the `pattern` should be extracted, or `0` for a static string. The `name` attribute
+is the key that will be reported in the case of a successful match and the `value`
+will either be a static string for `pos` values of `0` or missing and taken from the
+captured field.
+
+The `value` attribute supports interpolation of data from other fields. This is
```

Comment:
Kinda surprised we've had interpolation this long and didn't have it discussed in the docs.

Reply:
I agree! Thank you for adding that as a part of these changes.